backend/core/test/listings/listings-combined.e2e-spec.ts

```diff
@@ -385,8 +385,12 @@ describe("CombinedListings", () => {
         localListing.units.sort(sortUnits)
 
         // sort images
-        result.images.sort(sortImages)
-        localListing.images.sort(sortImages)
+        result.assets.sort(sortImages)
```

Comment:
This was needed to be changed because images are no longer getting populated on the returned object. The field is not being used anywhere and assets is what is used for getting images. So the original testing of this field was correct, but not required since the front-end doesn't care about it